source/hwIo/base.py

```diff
@@ -10,37 +10,43 @@
 See L{braille.BrailleDisplayDriver.isThreadSafe}.
 """
 
+from __future__ import annotations
 import sys
 import ctypes
 from ctypes import byref
 from ctypes.wintypes import DWORD
 from typing import Optional, Any, Union, Tuple, Callable
-
+import weakref
 import serial
 from serial.win32 import OVERLAPPED, FILE_FLAG_OVERLAPPED, INVALID_HANDLE_VALUE, ERROR_IO_PENDING, COMMTIMEOUTS, CreateFile, SetCommTimeouts
 import winKernel
 import braille
 from logHandler import log
 import config
 import time
+from .ioThread import IoThread, apcsWillBeStronglyReferenced
+# LPOVERLAPPED_COMPLETION_ROUTINE is imported for backwards compatibility.
+from .ioThread import LPOVERLAPPED_COMPLETION_ROUTINE  # NOQA: F401

```

Comment:
could this have the standard deprecation pattern - use `__getattr__` and log a warning?
https://github.com/nvaccess/nvda/blob/master/devDocs/deprecations.md